### PR TITLE
ES6

### PR DESCRIPTION
**Pull request by troutowicz**
ES6 and use tape rather than mocha for tests. Please review.

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -1,98 +1,3 @@
...
-.Trashes
-ehthumbs.db
-Thumbs.db
-
-
-
-##### node.js
-# Logs
-logs
-*.log
```
Comment (koresar) [inline .gitignore:15]: The `*.log` is necessary to ignore app crashes.

File: .gitignore [hunk 2]
```diff
@@ -1,98 +1,3 @@
...
 node_modules
-
-
-
-
-##### Covers JetBrains IDEs: IntelliJ, RubyMine, PhpStorm, AppCode, PyCharm, CLion
-*.iml
-
-## Directory-based project format:
-.idea/
```
Comment (koresar) [inline .gitignore:48]: Could you please put this line back? As I'm using this IDE. :) Thanks

File: README.md [hunk 3]
```diff
@@ -2,76 +2,91 @@
...
 ```
 
-# TODO
+## TODO
 * Tests
 * Travis
 * More `mixer.FUNCTION` functions
 
-# Want to contribute?
-It is Open Open Source. Whoever sends a PR, which gets accepted, receives the write permissions.
```
Comment (koresar) [inline README.md:77]: This is not a typo! It is "Open Open Source". As soon as I accept this PR you'll get write access to this repo. :)

**Comment by koresar** [comment]
That's a PR of my dream. Thank you very much! Except for those little issues I'm okay to merge it. :)

File: README.md (hunk 3 above)
Comment (koresar) [inline README.md:77]: Maybe "Open-Open Source" (dasherized) would sound better to confuse people less. :)

Comment (koresar) [inline README.md:77]: Does the following wording sound better?
`This is Open-Open Source project (yes, double open). Meaning that whoever sends a PR, which gets accepted, receives the write permissions.`

**Comment by troutowicz** [comment]
Changes made, and you're welcome!

**Comment by koresar** [comment]
You've just got the push access. :)

**Pull request merged**